frontend/src/components/modal.js

```diff
@@ -41,7 +41,8 @@ class ModalExample extends React.Component {
         packagerInfo["bundle"] = bundle
         // Add war object
         packagerInfo["war"] = {jenkinsVersion: this.state.warVersion}
-        packagerInfo["casc"] = true
+        // Make the casc section permanently false since no support for it
+        packagerInfo["casc"] = false
```

Comment:
You can also add a link the ticket for adding support for CasC support.

Reply:
Yeah sure sounds good :+1: